### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by jcromerohdz**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1595 @@
...
+                      Ocean as Its Toilet</h5>
+                    <p class="card-text">The director said deep sea research is "shamefully underfunded" and called for
+                      a "global fleet of swarm robotics" to study Earth's oceans. </p>
+                  </div>
+                </div>
+
+              </div>
+
+              <!-- TOP Story -->
+              <div class="col-lg-7 col-md-6 order-1 order-md-2 position-relative overflow-auto" id="top-story">
```
Comment (whiz25) [inline index.html:192]: Please use ```<article>``` tags for all sections containing articles.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1595 @@
...
+                  </div>
+                  <div class="card-body p-0 mt-2">
+                    <h5 class="card-title font-weight-bold h4">Trump's Call With Ukrainian President Dubbed 'Diplomatic
+                      Disaster' for Kiev</h5>
+                    <p class="card-text">The Kiev-based New Europe Center told Newsweek that Ukraine is now caught in a
+                      domestic U.S. political battle.</p>
+                  </div>
+                </div>
+                <!-- Culture Story -->
+                <div id="culture-story" class="position-relative">
```
Comment (whiz25) [inline index.html:214]: Wrap this section in ```article``` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1595 @@
...
+                    <h5 class="card-title font-weight-bold h4">James Cameron: Our So-Called Civilization Is Using the
+                      Ocean as Its Toilet</h5>
+                    <p class="card-text">The director said deep sea research is "shamefully underfunded" and called for
+                      a "global fleet of swarm robotics" to study Earth's oceans. </p>
+                  </div>
+                </div>
+
+              </div>
+
+              <!-- TOP Story -->
```
Comment (whiz25) [inline index.html]: remove all comments from your code

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1595 @@
...
+            <button class="btn" type="submit"></button>
+          </form>
+        </div>
+      </nav>
+
+      <div class="main container-fluid position-relative">
+        <div id="first-content" class="row mb-3">
+          <div class="col-lg-9 col-md-12">
+            <div class="row">
+              <!-- Features Story -->
```
Comment (whiz25) [inline index.html]: no comments

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good job :+1: @jcromerohdz 
Am impressed with your work so far. You have met most of the requirements so far. But we need to fix a few things before we can merge this one.
- [x] add ```.vscode``` folder to ```.gitignore```
- [x] remove the vertical scroll bar from the ```top story``` column. add a sticky effect instead.
- [x] make your ```navigation links navbar``` to span the whole width of the page so that the search icon is on the right most side of the navbar. make it's width 100%.
- [x] make your ```README``` more professional. add sections such as ```how to clone```, ```stack used```, ```author```,  etc
- [x] remove all comments in your code
## [OPTIONAL]
- [ ] use [githack](https://raw.githack.com/) for live version when the project hasn't been approved yet and ```git pages``` for approved project. This saves you from a lot headache.

Please make those changes and send for another review.

File: index.html (hunk 1 above)
Comment (jcromerohdz) [reply index.html:192]: Done

File: index.html (hunk 2 above)
Comment (jcromerohdz) [reply index.html:214]: Done

File: index.html (hunk 3 above)
Comment (jcromerohdz) [reply index.html]: Done

File: index.html (hunk 4 above)
Comment (jcromerohdz) [reply index.html]: Done

**Comment by jcromerohdz** [comment]
I try to put the .vscode folder in my .gitignore file but I don't why is not working, you can check my .gitignore file

**Review by uche-inyama — APPROVED** [review]
## Hi jcromerohdz,

This project is approved. :100: 

Go ahead and merge to master branch.

**Pull request merged**